src/features/auth/Login.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { loginUser } from "features";
+import { useNavigate, Link } from "react-router-dom";
+
+export const Login = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const token = useSelector(state=> state.auth.token);
+  const initialFormState={
+    username: "",
+    password:""}
+  const [loginCreds, setLoginCreds] = useState(initialFormState);
+  console.log(token);
+  const testLoginCreds = {
+    username: 'shikhasingh@gmail.com',
+    password: 'shikhasingh123'
```

Comment:
Suggestion - It is a good practice to add credentials/ keys in the .env file and then access them here.